src/centralized_telescope/setup.rs

```diff
@@ -0,0 +1,18 @@
+//! ALBA's Setup structure comprising, among others, the number of elements and internal parameters
+
+/// Setup output parameters
+#[derive(Debug, Clone, Copy)]
+pub struct Setup {
+    /// Approximate size of prover set to lower bound
+    pub set_size: u64,
+    /// Number of prover set's elements
+    pub proof_size: u64,
+    /// Maximum number of retries to find a proof
+    pub max_retries: u64,
+    /// Maximum number of subtrees to search to find a proof
+    pub search_width: u64,
+    /// Probability that a leaf was correctly chosen
```

Comment:
Now that I am thinking more and more about this name, we might be able to improve it even more as I'd be a bit confused about the term `leaf probability` if I am a first time reader. 

What is q: it is the probability that a random tuple `(t, s_1, \dots, s)u)` is a valid proof according to `h_2`. Maybe this can be said explicitly in the description to avoid confusion. Alternatively, it is the hardness. It sounds a bit more similar to `valid_proof_probability`?

Reply:
I think, giving an explicit description would solve the confusion. Since we have a dfs implementation, the name leaf probability seems appropriate but needs an explicit description of course.

Reply:
We chose `leaf_probability` as we really had in mind the trees of tuples and named several variables and functions accordingly (`H1 -> node_hash`, `H2 -> leaf_hash`, `d -> search_width` for the search arity).
You can see the random tuple as a candidate proof that you have to check, we considered it as the lead linked to a round that we were checking.

I'd suggest that we either add more descriptive comments here to explain succinctly what we are doing with each structure or change all the tree related names for consistency (`H1 -> round_hash`, `H2 -> proof_hash`, `d -> ?`).

Reply:
Updated